src/docfx/docset/Docset.cs

```diff
@@ -85,28 +85,28 @@ internal class Docset
         private readonly Lazy<RedirectionMap> _redirections;
         private readonly Lazy<LegacyTemplate> _legacyTemplate;
 
-        public Docset(Context context, string docsetPath, Config config, CommandLineOptions options)
-            : this(context, docsetPath, config, !string.IsNullOrEmpty(options.Locale) ? options.Locale.ToLowerInvariant() : config.Localization.DefaultLocale, options, null)
+        public Docset(Context context, string docsetPath, Config config, CommandLineOptions options, bool fallback = true)
```

Comment:
nit: maybe `fallback` -> `isDependency` to help express that we don't resolve loc for dependency docsets?